docs/content/components/dialog.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Dialog
+path: components/dialog
+status: Alpha
+source: 'https://github.com/github/github/tree/master/app/assets/stylesheets/experiments/modal-dialog.scss'
+bundle: dialog
+---
+
+Please note that the `<div>` element with `id="fake-container"` is not included in the component.
+
+```html live
+<div id="fake-container" style="height: 400px;">
+  <button class="btn"><span>Open dialog</span></button>
+  <div id="overlay-backdrop" class="Overlay-backdrop Overlay-backdrop--positionCenter Overlay-backdrop-positionWhenNarrow-fullScreen" role="dialog" aria-labelledby="title-id" aria-describedby="description-id" data-focus-trap="active">
+    <div class="Overlay Overlay--width-medium Overlay--height-medium Overlay--motion-scaleFade" data-focus-trap="active" open="">
```

Comment:
The demo is using a `--width-medium` modifier, but it's rendering full-width 🤔 :

> ![Output rendering](https://user-images.githubusercontent.com/293280/165398783-0b767540-fa1c-4b4d-a925-df417776d15a.png)

Reply:
I realized this demo is using old classed 😄 fixed